File: src/main/java/com/kingdom/model/computer/EasyComputerPlayer.java
package com.kingdom.model.computer;

import com.kingdom.model.Card;
import com.kingdom.model.Game;
import com.kingdom.model.Player;

public class EasyComputerPlayer extends ComputerPlayer {
    public EasyComputerPlayer(Player player, Game game) {
        super(player, game);
        difficulty = 1;
    }

    @Override
    protected void setupStartingStrategies() {
    }

    @Override
    protected boolean excludeCard(Card card) {
        return excludeCardEasy(card);
    }
}

File: src/main/kotlin/com/kingdom/model/computer/EasyComputerPlayer.kt
package com.kingdom.model.computer

import com.kingdom.model.cards.Card
import com.kingdom.model.OldGame
import com.kingdom.model.OldPlayer

class EasyComputerPlayer(player: OldPlayer, game: OldGame) : ComputerPlayer(player, game) {
    init {
        difficulty = 1
    }

    override fun setupStartingStrategies() {}

    override fun excludeCard(card: Card): Boolean {
        return excludeCardEasy(card)
    }
}
